File: guitar/src/app/model/orderItem.ts
export class OrderItem {

    //Table ID
    id: number = 0;

    //User Data
    orderId: number = 0;
    userId: number = 0;
    userName?: string = "";

    //Product Data
    productId: number = 0;
    productName: string = "";
    imageFile: string = "";
    quantity: number = 0;
    price: string = "0.0";
  
    //SKU??

    //status
    status: string ="not_selected";

    //For UI
    showConfirm?: boolean; //For UI
}

File: src/main/java/com/rosal/model/OrderItem.java
package com.rosal.model;
import lombok.Data;

@Data
public class OrderItem {

    //User Data
    int id;
   // int orderId;
   int userId;
    String userName;
    int orderId;

    //Product Data
    int productId;
    String productName;
    String imageFile;
    double quantity;
    double price;
    String status;

}

